packages/sentinel/src/models/networks.ts

```diff
@@ -0,0 +1 @@
+export type NetworkType = 'prod' | 'test';
```

Comment:
nits eventualy would use full words like "production" if not "mainnet" mentioned by Ze